playwright-gh-pages/deploy-report-pages/build-pr-comment.js

```diff
@@ -105,29 +106,14 @@ async function buildPrComment() {
   // Add sorted rows to table
   table += '\n' + rows.join('\n') + '\n';
 
+  const ciLink = `https://github.com/${process.env.GITHUB_REPOSITORY_OWNER}/${process.env.GITHUB_REPOSITORY_NAME}/blob/main/.github/workflows/ci.yml`;
```

Comment:
you can't assume there's a `main` branch. it might be a `master` branch. try using `github.event.repository.default_branch`  or see if you can find here https://docs.github.com/en/webhooks/webhook-events-and-payloads#repository

Reply:
fixed in https://github.com/grafana/plugin-actions/pull/52/commits/ef27477b2fb904386461b2c47f192be8228622d9